.github/workflows/tests-datapath-verifier.yaml

```diff
@@ -113,11 +113,19 @@ jobs:
 
             git config --global --add safe.directory /host
             uname -a
+
             # The LVH image might ship with an arbitrary Go toolchain version,
             # install the same Go toolchain version as current HEAD.
             CGO_ENABLED=0 GOPROXY=direct GOSUMDB= go install golang.org/dl/go${{ env.go-version }}@latest
             go${{ env.go-version }} download
 
+            # The LVH image ships with LLVM taken from a release Cilium version.
+            # Replace it with the one extracted from the cilium-builder image.
+            /bootstrap/deb-docker.sh # Install docker first.
+            /host/contrib/scripts/extract-llvm.sh /tmp/llvm
+            mv /tmp/llvm/usr/local/bin/{clang,llc} /bin/
+            rm -r /tmp/llvm
+
```

Comment:
This would basically make the logic in https://github.com/cilium/little-vm-helper-images/blob/main/_data/bootstrap/bpf-tools.sh obsolete, right?

Reply:
It'd be nice to embed docker in there, we don't need that to be dynamic. The changes with this workflow seem like they take about 1-2 minutes longer than the current time for this workflow in main, so we could probably reduce that time further that way. 

(The workflow already finishes in <10m anyway so not critical, but making it faster is always good as a follow up :rocket: )